terraform/monitoring/02_dashboards.tf

```diff
@@ -114,4 +114,11 @@ resource "google_monitoring_dashboard" "userexp_dashboard" {
   }
 }
 EOF
-}
\ No newline at end of file
+}
+
+# This resource creates dashboards for the frontend service of the
+# Stackdriver Sandbox. The JSON object containing the exact details
+# of the dashboard can be found in the 'dashboards' folder.
+resource "google_monitoring_dashboard" "frontend_dashboard" {
+  dashboard_json = file("./dashboards/frontend_dashboard.json")
```

Comment:
I might suggest moving the userexp_dashboard to it's own file as well.  You should be consistent on either doing it inline or with files.

Reply:
Changed :)